WGS_pipeline/out/small.vcf.status.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
should this and/or small.vcf.err be public?

Reply:
sorry..